src/Altinn.App.Core/Infrastructure/Clients/Authorization/AuthorizationClient.cs

```diff
@@ -179,4 +182,121 @@ List<string> actions
         }
         return MultiDecisionHelper.ValidatePdpMultiDecision(actionsResult, response.Response, user);
     }
+
+    /// <inheritdoc />
+    public async Task<List<string>> GetKeyRoleOrganisationParties(int userId, List<string> orgNumbers)
+    {
+        XacmlJsonRequestRoot request = CreateXacmlJsonRequest(userId, orgNumbers);
+        XacmlJsonResponse response = await _pdp.GetDecisionForRequest(request);
+
+        if (response?.Response == null)
+        {
+            return [];
+        }
+
+        List<string> organisations =
+        [
+            .. response
+                .Response.Where(result => result.Decision == "Permit")
+                .SelectMany(result => result.Category)
+                .SelectMany(category => category.Attribute)
+                .Where(attribute => orgNumbers.Contains(attribute.Value))
+                .Select(attribute => attribute.Value),
+        ];
+
+        return organisations;
+    }
+
```

Comment:
Kanskje skrive en kommentar om hva i alle dager dette er for noe. :P Men kan evt. gjøres i egen PR senere også.

Reply:
Altså, s1, a1 osv. At authorization lagde noe special for oss.